includes/class-gp-translation-helpers.php

```diff
@@ -198,7 +314,7 @@ public function css_and_js() {
 				$js = $translation_helper->get_js();
 				if ( $js ) {
 					echo '/* Translation Helper:  ' . esc_js( $translation_helper->get_title() ) . ' */' . "\n";
-					echo $js . "\n";
+					echo $js . "\n"; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
```suggestion
					echo esc_js( $js ) . "\n";
```

Reply:
Oh, wow. Could you give `esc_html()` a try?

Reply:
It breaks the code, too, because it escapes the single quotes.

Here is the code working fine.

![image](https://user-images.githubusercontent.com/1667814/152973974-6e9d6ddd-7f95-4cc6-9872-58846ecef6d6.png)

And here with the esc_html() function:

![image](https://user-images.githubusercontent.com/1667814/152974123-2a9bdb15-7931-47b7-97e2-790c19c90466.png)

![image](https://user-images.githubusercontent.com/1667814/152974203-96130c59-15c3-4ad3-824b-c98d268c68f8.png)

It works if I replace the '&#039;' with a single quote:

![image](https://user-images.githubusercontent.com/1667814/152974639-cf42f147-e5ec-4142-a5f1-501dd2407a57.png)

Reply:
With the str_replace() function, the PHPCS doesn't work, because the code prints an unescaped string, so we have the same problem.

Reply:
Ok then let's keep it this way.